docs/internals/optimizer.rst

```diff
@@ -1263,7 +1263,7 @@ Reverses the transformation of ForLoopConditionIntoBody.
 
 For any movable ``c``, it turns
 
-::
+.. code-block:: none
 
     for { ... } 1 { ... } {
     if iszero(c) { break }
```

Comment:
I tried to use `yul` with `:force:` for this but I can't because we try to compile all yul snippets in `cmdlineTests.sh` and these obviously won't compile.